cmake/pcl_all_in_one_installer.cmake

```diff
@@ -26,16 +26,16 @@ if(BUILD_all_in_one_installer)
         list(APPEND PCL_3RDPARTY_COMPONENTS ${dep})
     endforeach(dep)
 
-    if(WITH_REALSENSE2)
-        get_filename_component(REALSENSE2_ROOT "@REALSENSE2_INCLUDE_DIRS@" PATH)
+    if(WITH_RSSDK2)
+        get_filename_component(RSSDK2_ROOT "@RSSDK2_INCLUDE_DIRS@" PATH)
         install(
-                DIRECTORY "${REALSENSE2_ROOT}/"
-                DESTINATION 3rdParty/librealsense2
-                COMPONENT librealsense2
```

Comment:
@UnaNancyOwen -- I believe I updated these values correctly to handle the rename (realsense2 -> RSSDK2), can you please review?

One notable update here -- I changed DESTINATION from `3rdParty/librealsense2` -> `3rdParty/librealsense` (without the trailing 2), which I _think_ is more correct, and inline with the default installation directory for librealsense.

It is a small distinction that RSSDK 2.0 = librealsense (with no 2)... I think it could cause headaches later if they ever release an actual librealsense 2.0, and we were already calling it librealsense2, but please let me know if you think this is appropriate.

Reply:
I was put "2" in here, because the name of default installation directory is "librealsense2" (e.g. <code>C:/Program Files/librealsense2</code>). I think it would be better to follow it.
And, librealsense1 and librealsense2 (= RealSense SDK 2.0) is different things. I think it will not misunderstood.

If you change to <code>3rdParty/librealsense</code>, It is necessary to change PCLConfig.cmake.in#L217.

```patch
  if(PCL_ALL_IN_ONE_INSTALLER)
-    set(realsense2_DIR "${PCL_ROOT}/3rdParty/librealsense2/lib/cmake/realsense2" CACHE PATH "The directory containing realsense2Config.cmake")
+    set(realsense2_DIR "${PCL_ROOT}/3rdParty/librealsense/lib/cmake/realsense2" CACHE PATH "The directory containing realsense2Config.cmake")
  elseif(NOT realsense2_DIR)
```

What do you think?

Reply:
>  I think it could cause headaches later if they ever release an actual librealsense 2.0, and we were already calling it librealsense2, but please let me know if you think this is appropriate.

This **is** actual `librealsense 2.0` and it is not compatible with the old ("1") version, and it is installed by default to "librealsense2" folder, just as Tsukasa wrote.

Reply:
My mistake! I'll change this back to librealsense2